### PR TITLE
Buyankhuu/orders

### PR DESCRIPTION
**Pull request by BuyankhuuTsCAl**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
- added the order page 
### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy!"
https://www.loom.com/share/71467fe443864e6485272df2a3e01b4e?sid=e35f054b-3f85-475e-aaab-32a9971bea66

## How to review
[//]: # "Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?"
- once you go to any page, press the cart button which will take you to the order page. 
- scroll to see how I handled the order flow 
- none of the buttons work atm 

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."
- The backend is not done 
- the data is from favorites 
### Online sources
[//]: # "Optional - copy links to any tutorials or documentation that was useful to you when working on this PR"

### Related PRs
[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

[//]: # "This tags the project leader as a default. Feel free to change, or add on anyone who you should be in on the conversation."
🛍 CC: @EthanAuyeung

### CODE REVIEW

File: src/app/cart/style.tsx
Comment (EthanAuyeung) [inline src/app/cart/style.tsx:1]: Rename to styles.ts

File: src/app/profileScreen/style.ts
Comment (EthanAuyeung) [inline src/app/profileScreen/style.ts:1]: Also rename to styles.ts

File: src/app/login/shantiLogo.png
Comment (EthanAuyeung) [inline src/app/login/shantiLogo.png:1]: Is there a way to put this in the styles folder or an image folder somewhere?

Comment (BuyankhuuTsCAl) [reply src/app/login/shantiLogo.png:1]: wait that's not on my local.

Comment (BuyankhuuTsCAl) [reply src/app/login/shantiLogo.png:1]: it's already in the public/image folder since that's the only way to render images i think?

**Pull request merged**